### ISSUE TITLE
KeyError when using postgraphql

### ISSUE THREAD

**Issue by kepi** [perseas/Pyrseas#180]
I'm getting `KeyError: ('public', 'postgraphql_watch.notify_watchers', '')` on empty database watched by PostgraphQL.

I'm not sure what is the issue as there is no error specified, only KeyError.

Here is script which is initializing databaze for PostgraphQL:
https://github.com/postgraphql/postgraphql/blob/master/resources/watch-fixtures.sql

Rest is empty database on PostgreSQL 10.1 (same problem was on 9.6).

I'm running `dbtoyaml` with ` dbtoyaml -H localhost -p ${PG_PORT} -U postgres mydb`, I also tried to add ` -N postgraphql_watch` but problem is probably with function defined in `public` schema.

```
Traceback (most recent call last):                                                                                                                                                                                                                            
  File "/home/kepi/.local/bin/dbtoyaml", line 11, in <module>
    sys.exit(main())
  File "/home/kepi/.local/lib/python2.7/site-packages/pyrseas/dbtoyaml.py", line 49, in main
    dbmap = db.to_map()
  File "/home/kepi/.local/lib/python2.7/site-packages/pyrseas/database.py", line 442, in to_map
    self.from_catalog(True)
  File "/home/kepi/.local/lib/python2.7/site-packages/pyrseas/database.py", line 352, in from_catalog
    self._link_refs(self.db)
  File "/home/kepi/.local/lib/python2.7/site-packages/pyrseas/database.py", line 224, in _link_refs
    db.functions.link_refs(db.types, db.eventtrigs)
  File "/home/kepi/.local/lib/python2.7/site-packages/pyrseas/dbobject/function.py", line 667, in link_refs
    func = self[(sch, fnc[:-2], '')]                                                                                                                                                                                                                          
KeyError: ('public', 'postgraphql_watch.notify_watchers', '')                          
```

**Comment by jmafc** [comment]
This appears to be the same problem as discussed in #165. We're also in the process of not treating the `public` schema a special case (see #176), so at least we'll get a better error message.

**Comment by jmafc** [comment]
@kepi I have installed Postgraphile and used it against a test database (with `--watch` which appears to be needed to generate something like your scenario). I then ran `dbtoyaml` against that database without any errors. It lists two event triggers, the `postgraphile_watch` schema and two functions: `notify_watchers_ddl()` and `notify_watchers_drop()`. Thus, I believe this problem has been fixed in the master branch. If I don't hear from you in a week or so, I'll close the issue.
